index.bs

```diff
@@ -10467,177 +10467,195 @@ asynchronous sections can't be reordered.
+				1. If this {{AudioParam}} has any {{AudioNode}} connected to it,
+					<a href="#SummingJunction">sum</a> the buffers
+					<a href="#available-for-reading"> made available for reading</a> by
+					all {{AudioNode}} connected to this {{AudioParam}},
+					<a href="#down-mix">down mix</a> the resulting buffer down to mono
+					channel, and call this buffer the <dfn id="input-audioparam-buffer">
+					input AudioParam buffer</dfn>.
 
-	7. <a href="#computation-of-value">Compute the value(s)</a> of the
-		{{AudioListener}}'s {{AudioParam}}s for this block.
+				2. <a href="#computation-of-value">Compute the value(s)</a> of this
+					{{AudioParam}} for this block.
 
-	8. For each {{AudioNode}}, in the order determined previously:
-		1. For each {{AudioParam}} of this {{AudioNode}}, execute
-			these steps:
+				3. <a>Queue a control message</a> to set the {{[[current value]]}} slot
+					of this {{AudioParam}} to the last value computed in the preceding
+					step.
 
-			1. If this {{AudioParam}} has any {{AudioNode}}
-				connected to it, <a href="#SummingJunction">sum</a> the
-				buffers <a href="#available">made available for reading</a>
-				by all {{AudioNode}} connected to this {{AudioParam}},
-				<a href="#down-mix">down mix</a> the resulting buffer down to
-				Mono, and call this buffer the <dfn id="input-audioparam-buffer">input AudioParam
-				buffer</dfn>.
+			2. If this {{AudioNode}} has any {{AudioNode}}s connected to its input,
+				<a href="#SummingJunction">sum</a> the buffers
+				<a href="#available-for-reading">made available for reading</a> by all
+				{{AudioNode}}s connected to this {{AudioNode}}. The resulting buffer is
+				called the <dfn>input buffer</dfn>.
+				<a href="#channel-up-mixing-and-down-mixing">Up or down-mix</a> it to
+				match if number of input channels of this {{AudioNode}}.
 
-			2. <a href="#computation-of-value">Compute the value(s)</a> of
-				this {{AudioParam}} for this block.
+			3. If this {{AudioNode}} is a <a>source node</a>,
+				<a href="computing-block"> compute a block of audio</a>, and
+				<a href="#available-for-reading">make it available for reading</a>.
 
-			3. <a>Queue a control message</a> to set the
-				{{[[current value]]}} slot of this {{AudioParam}}
-				to the last value computed in the preceding step.
+			4. Else, if this {{AudioNode}} is a <a>destination node</a>,
+				<a href="record-input">record the input</a> of this {{AudioNode}}.
 
-		2. If this {{AudioNode}} has any {{AudioNode}}s connected
-			to its input, <a href="#SummingJunction">sum</a> the buffers
-			<a href="#available">made available for reading</a> by all
-			{{AudioNode}}s connected to this {{AudioNode}}. The
-			resulting buffer is called the <dfn>input buffer</dfn>.
-			<a href="#channel-up-mixing-and-down-mixing">Up or down-mix</a>
-			it to match if number of input channels of this
-			{{AudioNode}}.
+		  5. If this {{AudioNode}} is an {{AudioWorkletNode}}, invoke associated
```

Comment:
I think this should be indented one more level?  That is part of the "For each AudioNode" loop.